build.gradle.kts

```diff
@@ -122,6 +123,45 @@ val bootstrapJar = tasks.register<Copy>("bootstrapJar") {
     into(file("$buildDir/processedResources/jvm/main"))
 }
 
+// Below are tasks that are used by the trace debugger plugin.
```

Comment:
Is it possible to move all the code related to Trace Debugger to a separate `kts` file?

Reply:
@dmitrii-artuhov here is an example in the Kotlin Coroutines repo: 
https://github.com/Kotlin/kotlinx.coroutines/blob/master/buildSrc/src/main/kotlin/Publishing.kt